scripts/mineBlocks.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async () => {
+  const [times] = await process.argv.slice(4)
+
+  const mineOneBlock = async (id) => web3.currentProvider.send({
+    jsonrpc: "2.0",
+    method: "evm_mine",
+    params: [],
+    id,
+  }, (err, res) => !err ? console.log(res) : err )
+
+  for (let i = 0; i < times; ++i) {
+    await mineOneBlock(i)
+  }
+  console.log("mined", times, "blocks")
+}
```

Comment:
So should we throw await our wait 20 block function in favour of this one?

Reply:
There is already a `waitForNBlocks` function in utilities.js. Can we consolidate?

Reply:
Yes, we should.

I will make an issue